examples/zephyr/common/Kconfig.defconfig

```diff
@@ -0,0 +1,101 @@
+#
+# Copyright (C) 2022-2023 Golioth, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if GOLIOTH_SAMPLE_COMMON
+
+if DNS_RESOLVER
+
+config DNS_SERVER_IP_ADDRESSES
+	default y
+
+config DNS_SERVER1
+	default "1.1.1.1"
+
+endif # DNS_RESOLVER
+
+if MBEDTLS
+
+config MBEDTLS_ENABLE_HEAP
+	default y
+
+if MBEDTLS_ENABLE_HEAP
+
+config MBEDTLS_HEAP_SIZE
+	default 49152 if GOLIOTH_AUTH_METHOD_CERT && NRF_SECURITY
+	default 32768 if GOLIOTH_AUTH_METHOD_CERT
+	default 10240
+
+endif # MBEDTLS_ENABLE_HEAP
+
+if MBEDTLS_BUILTIN
+
+config MBEDTLS_SSL_MAX_CONTENT_LEN
+	default 4096 if GOLIOTH_AUTH_METHOD_CERT
+	default 2048
+
+endif # MBEDTLS_BUILTIN
+
+if NRF_SECURITY
+
+config MBEDTLS_SSL_IN_CONTENT_LEN
+	int
+	default 4096 if GOLIOTH_AUTH_METHOD_CERT
+	default 2048
+
+config MBEDTLS_SSL_OUT_CONTENT_LEN
+	int
+	default 4096 if GOLIOTH_AUTH_METHOD_CERT
+	default 2048
+
+endif # NRF_SECURITY
+
+endif # MBEDTLS
+
+config NETWORKING
+	default y
+
+config NET_IPV4
+	default y
+
+config NET_IPV6
+	default n
+
+config NET_LOG
+	default y if NETWORKING
+
+config NET_SHELL
+	default y if NETWORKING
+
+config MAIN_STACK_SIZE
+	default 4096
+
+config MINIMAL_LIBC_MALLOC_ARENA_SIZE
+	default 2048 if MINIMAL_LIBC
+
+config LOG_PROCESS_THREAD_STACK_SIZE
+	default 2048 if LOG_BACKEND_GOLIOTH
```

Comment:
There is no Golioth log backend yet, so we don't need this. Are there other settings in this file that go beyond what we need for these samples?

Reply:
Ah, good point. I've removed everything form defconfig that wasn't directly used in the current state of golioth_basics.